portfolio/src/main/webapp/script.js

```diff
@@ -194,6 +189,10 @@ function getUserLocation() {
         });
         marker.setMap(map);
         map.setCenter(userLocation);
+
+        var circle = new google.maps.Circle(
+          {center: userLocation, radius: 5000});
```

Comment:
5000 is a magic number. Is it miles, steps, meters, etc? Don't use magic numbers and make it a const (add `const LOCATION_LIMIT_MILES` or a similar name with the unit near the top of the file)

Reply:
Noted, thank you, thank you for the feedback, Yi-Tao!